_implementors/adapters.md

```diff
@@ -37,7 +37,7 @@ The following table lists the known debug adapters that implement the Debug Adap
 [ESP32 Debugger](https://github.com/espressif/esp-debug-adapter)|[@espressif](https://github.com/espressif)|[esp_debug_adapter](https://github.com/espressif/esp-debug-adapter)
 [Firefox Remote Debug](https://marketplace.visualstudio.com/items?itemName=yurydelendik.firefox-debug)|[@yurydelendik](https://github.com/yurydelendik)|[yurydelendik/vscode-ff-debug](https://github.com/yurydelendik/vscode-ff-debug)
 [Flash](https://marketplace.visualstudio.com/items?itemName=vshaxe.haxe-debug)|[@vshaxe](https://github.com/vshaxe/)|[vshaxe/flash-debugger](https://github.com/vshaxe/flash-debugger)
-[Go](https://marketplace.visualstudio.com/items?itemName=golang.Go)|[@golang/tools-team](https://github.com/orgs/golang/teams/tools-team)|[golang/vscode-go](https://github.com/golang/vscode-go)
+[Go](https://github.com/go-delve/delve/tree/master/Documentation/api/dap)|[Delve](https://github.com/go-delve)|[go-delve/delve](https://github.com/go-delve/delve)

```

Comment:
Based on how other entries look, I think this might need to say @polinasok and Delve.

Reply:
Didn' realize you were not part of delve org. Edited.